test/models/country_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class CountryTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
```

Comment:
I recommend some tests to ensure that the validations run and work. I would expect them to look like the model tests in [Lesson 4.2](https://learn.codethedream.org/rails-basic-skills-rspec/).